web/modules/custom/feedback/src/Form/FeedbackForm.php

```diff
@@ -1,187 +0,0 @@
-    $record = array();
-    if (isset($getFilterValue)) {
-      $query = $this->database->select('feedback', 'f')
-        ->condition('feedback_id', $getFilterValue)
-        ->fields('f');
-      $record = $query->execute()->fetchAssoc();
-    }
-
-    $form['name'] = array(
-      '#title' => $this->t('Your Name'),
-      '#type' => 'textfield',
-      '#size' => 50,
-      '#required' => TRUE,
-      '#default_value' => (isset($record['name']) && $getFilterValue) ? $record['name']:'',
-    );
-    $form['email'] = array(
-      '#title' => $this->t('Your Email'),
-      '#type' => 'email',
-      '#size' => 25,
-      '#required' => TRUE,
-      '#default_value' => (isset($record['email']) && $getFilterValue) ? $record['email']:'',
-    );
-    $form['message'] = array(
-      '#title' => $this->t('Your Message'),
-      '#type' => 'textarea',
-      '#required' => TRUE,
-      '#default_value' => (isset($record['message']) && $getFilterValue) ? $record['message']:'',
-    );
-    $form['actions']['#type'] = 'actions';
-    $form['actions']['submit'] = array(
-      '#type' => 'submit',
-      '#value' => $this->t('Send')
-    );
-    return $form;
-  }
-
-  /**
-   * start validation forms
-   * @inheritDoc
-   */
-  public function validateForm(array &$form, FormStateInterface $form_state)
-  {
-    if (strlen($form_state->getValue('name')) < 5) {
-      $form_state->setErrorByName('name', $this->t('Name is too short.'));
-    }
-    if (strlen($form_state->getValue('message')) < 1) {
-      $form_state->setErrorByName('message', $this->t('Message is too short.'));
-    }
-
-    $config = $this->configFactory->getEditable('feedback.settings');
-    $configuration = $config->get('allowed_value');
-//    get user's email
-    $email_now = $form_state->getValue('email');
-//    check email in the database
-    $query = $this->database->select('feedback', 'f');
-    $query->addField('f', 'email');
-    $email_db = $query->execute()->fetchCol();
-
-    if ($email_db[0] == $email_now && $configuration == 1 ) {
-      $form_state->setErrorByName('email', $this -> t('The email can not be use for form resubmit'));
-    }
-  }
-
-  /**
-   * Start submit form
-   * @inheritDoc
-   */
-  public function submitForm(array &$form, FormStateInterface $form_state) {
-    $getFilterValue = $this->requestStack->getCurrentRequest()->query->get('num');
-    if (isset($getFilterValue)) {
-      $query = $this->database;
-      $userId = $this->currentUser->id();
-      $user = \Drupal\user\Entity\User::load($this->currentUser->id());
-      $query->update('feedback')
-        ->fields([
-          'name' => $form_state -> getValue('name'),
-          'email' => $form_state -> getValue('email'),
-          'message' => $form_state -> getValue('message'),
-          'uid' => $user -> id(),
-          'created' => time(),
-        ])
-        ->condition('feedback_id', $getFilterValue)
-        ->execute();
-      drupal_set_message("Successfully changed");
```

Comment:
1. use single quotes where possible
2. use $this->t() to be sure all texts will be translatable